pkg/activator/net/throttler.go

```diff
@@ -634,3 +651,5 @@ func (ib *infiniteBreaker) Maybe(ctx context.Context, thunk func()) error {
 		return ctx.Err()
 	}
 }
+
+func (ib *infiniteBreaker) HasCapacity() bool { return true }
```

Comment:
shouldn't this be `return ib.Capacity() > 0`?

Reply:
I guess I presumed we're within the outer breaker. Done.